Xamarin.Forms.Maps/Polyline.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Text;
+
+namespace Xamarin.Forms.Maps
+{
+	public class Polyline : BindableObject
+	{
+		public static readonly BindableProperty StrokeColorProperty = BindableProperty.Create(
+			nameof(StrokeColor),
+			typeof(Color), 
+			typeof(Polyline),
+			Color.Default);
+
+		public static readonly BindableProperty StrokeWidthProperty = BindableProperty.Create(
+			nameof(StrokeWidth), 
+			typeof(float), 
+			typeof(Polyline), 
+			5f);
+
+		public Color StrokeColor
+		{
+			get => (Color)GetValue(StrokeColorProperty);
+			set => SetValue(StrokeColorProperty, value);
+		}
+
+		public float StrokeWidth
+		{
+			get => (float)GetValue(StrokeWidthProperty);
+			set => SetValue(StrokeWidthProperty, value);
+		}
+
+		public IList<Position> Geopath { get; }
+
+		[EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Why the `EditorBrowsableState.Never` for `PolylineId`?

Reply:
It's a analogue to `Pin.MarkerId`, which also has the attribute. Something for the renderers to match the cross-platform object to its native counterpart, shouldn't really be used outside of that.

Reply:
Could it be made `internal`?

Reply:
Only if you're okay with adding an `InternalsVisibleToAttribute` back to the Maps project. This property is defined in the shared project and used in the renderers.

Reply:
Argh. Good point. `public` it is, then.